editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
```

Comment:
It always comes out as `undefined` because `isFirst` is `undefined`. JSX conversion will interpret the value of `onClick` to be `undefined` regardless of what is on the other side of `?` in our ternary. Do you want to pass in `isFirst` and `isLast` as `false` for each, I added another set of test cases to accomplish what I think you are looking for?